docs/linking/4-grant-consent.puml

```diff
@@ -47,5 +51,36 @@ Switch -> PISP ++: ""POST /consents""\n\
     ""}""
 
 PISP --> Switch: ""202 Accepted""
+deactivate Switch
+deactivate PISP
+
+== Failure case ==
+
+DFSP -> DFSP: Unable to fetch stored scopes
```

Comment:
I'm not sure I understand this scenario. Maybe if you split it into it's own sequence diagram, that responds with a `PUT /consentRequests/456/error` to a `PATCH /consentRequests/456` after some generic processing error on the DFSP side, that would make more sense.